openshift/4.0/templates/api/build.yaml

```diff
@@ -20,6 +20,16 @@ parameters:
     description: The name of the application component role (e.g ui, web, api, database).
     required: true
     value: api
+  - name: PROJECT_NAMESPACE
```

Comment:
Some cleanup and applying some consistent standards across templates.  Not related to this PR, but doesn't particular hurt to include it here either.